x-pack/plugins/endpoint/server/services/bootstrap/index.ts

```diff
@@ -0,0 +1,249 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { IScopedClusterClient, LoggerFactory, Logger } from 'kibana/server';
+import { alreadyExists } from './errors';
+// figure out how to read in a json file
+import endpointTemplate from './template.json';
+
+const templateVersion = 1;
+const appName = 'endpoint';
+const ilmName = 'endpoint_policy';
+const fanPipeline = 'route-events';
+const alertsIndex = 'alerts';
+
+// TODO need to get the version of the endpoint app
+const appVersion = '1.0.0';
+
+const defaultILM = {
+  policy: {
+    phases: {
+      hot: {
+        actions: {
+          rollover: {
+            max_size: '50GB',
+            max_age: '30d',
+          },
+        },
+      },
+      delete: {
+        min_age: '90d',
+        actions: {
+          delete: {},
+        },
+      },
+    },
+  },
+};
+
```

Comment:
consider if we have to make things more visible using the config with defaults. https://github.com/elastic/kibana/blob/endpoint-security-poc/x-pack/plugins/endpoint/server/config.ts